Samples/Interactivity/Commands/InteractiveCommands.cs

```diff
@@ -115,7 +118,7 @@ public async Task<IResult> SendDropdownAsync()
             {
                 new SelectMenuComponent
                 (
-                    "colour-dropdown",
+                    $"{Constants.InteractionTree}::{Constants.SelectMenuPrefix}::colour-dropdown",
```

Comment:
I would actually like it if ``SelectMenuComponent`` prefixed this for us instead so we do not have to think about this ourselves on remembering to prefix it.

Reply:
I do not think it would be useful to have SelectMenuComponent do it as it is a base API object and should not be quite opinionated.
As well to have it not prefix so also other interactivity systems could be combined.

Reply:
Agreed; this rework is intended to allow coexistence with other systems.